libs/api/observability/src/lib/services/pino-logger.service.ts

```diff
@@ -1,4 +1,6 @@
 import pino from 'pino';
+// import pino-pretty to make dependency explicit to NX
+import 'pino-pretty';
```

Comment:
move this into the constructor like this:

	```
if (debug) {
			import('pino-pretty');
}
```

nx will still include pino-prettier in the package.json, but the module is not loading when it is not needed.

Reply:
https://github.com/kordis-leitstelle/kordis/pull/326/commits/038e6923b8764f301d6012060844801f7336e1ec